CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 ## Contributing
 
-Chat with us in the [bugbug](https://chat.mozilla.org/#/room/#bugbug:mozilla.org) Matrix room.
-
-1. [Issues marked as `good-first-bug`](https://github.com/mozilla/bugbug/labels/good-first-bug) are self-contained enough that a contributor should be able to work on them.
-2. Issues are considered not assigned, until there is a PR linked to them. Feel free to work on any unassigned issue, you don't need to ask first.
+Steps to contribute a project on Github for beginners:
+1. Create a fork of the project suitable for you on Github.This will create a personal repository on your computer.
+2. Clone the project on your machine and create a branch from master to start working on the project.
+3. Go through the project and make the commits as per the requirements,fix the code/bugs and test your features.
+4. Push the commits so that the project owner could implement the features corrected by you.
+5. Make a pull request on Github.
```

Comment:
Now we can resume from here: 
11. Make a pull request on Github. You can follow this [guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) on how to make a pull request from a forked repository.

Reply:
### Submitting your Pull Request(PR)
The PR needs to be approved by the project maintainers. In order to ease the review process, it is recommended you follow these principles:
- Let the PR have a helpful title.
- Link your PR to the issue it is solving, closing or resolving using `Fixes/Closes/Resolves  #PRNumber`
- Ensure your changes have produced no code style or linting issues. `pre-commit` will run these checks. If any issues are found, the tests will fail.